.ci/manywheel/build_rocm.sh

```diff
@@ -252,6 +252,20 @@ RCCL_SHARE_FILES=($(ls $RCCL_SHARE_SRC))
 DEPS_AUX_SRCLIST+=(${RCCL_SHARE_FILES[@]/#/$RCCL_SHARE_SRC/})
 DEPS_AUX_DSTLIST+=(${RCCL_SHARE_FILES[@]/#/$RCCL_SHARE_DST/})
 
+# PyTorch 2.6+ (AOTriton 0.8b+)
```

Comment:
Manywheel related code to add GPU kernel files (which now shipped separated to ensure libaotriton_v2.so is not too bloated)